src/components/atoms/badge/badge.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { Badge, BadgeProps } from "theme-ui";
+
+export default {
+    title: "atoms/Badge",
+    component: Badge,
+};
+
+export const subtleBadge = (props: BadgeProps) => <Badge variant="test">TEST</Badge>
```

Comment:
Is this supposed to have the `test` variant?

Reply:
Not really, used test variant to check if I could see the styling component in storyboard, but i cant and i dont know why.